charts/cronjob/values.yaml

```diff
@@ -21,6 +21,11 @@ healthCheck:
   healthCheckCommand: ["cat", "/tmp/health"]
   initialDelaySeconds: 0
 
+concurrencyPolicy:
+  value: "Allow"
```

Comment:
IMO there is no need for nested "value"

Simply
```
concurrencyPolicy: "Allow"
```

Also, can you please set Forbid